ibis/expr/datatypes/core.py

```diff
@@ -678,6 +678,12 @@ def types(self) -> tuple[DataType, ...]:
         """Return the types of the struct's fields."""
         return tuple(self.fields.values())
```

Comment:
This could be `self.values()` now.

Reply:
Similarly with `names` -> `self.keys()`.

Reply:
Addressing that in the upcoming PR.

Reply:
Or I can update here too since I need to push again.

Reply:
Either way works!

Reply:
The build issue should be resolved once all the PRs get merged, so I'm not updating this PR.